O/openal/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "openal"
```

Comment:
Ok, last comment: maybe call it [`OpenAL`](https://github.com/kcat/openal-soft)?
```suggestion
name = "OpenAL"
```
You need to rename also the directory accordingly